### DISCUSSION TITLE
Links inside Discussions comment replies on mobile can’t be tapped on

### DISCUSSION THREAD

**Discussion by ghost** [community/community#2866] [Mobile] [answered]
![image](https://user-images.githubusercontent.com/65474198/101562310-6bae8d80-3984-11eb-83fc-d483bfffc990.jpeg)

I tried clicking on the blue link there but nothing happened. Also, on those replies there is no three dot menu, so you can't do something like edit your reply until someone else replies and you can view the whole thread, there you can find three dot menus.

**Comment by Iron-Ham** [comment, marked as answer]
Hi @BenSova thanks for bringing this to our attention. 

We should be releasing a fix for link-support in inline replies shortly. I've taken note of your feedback regarding the three-dot button, and I've also noted that users cannot react to inline replies, and will be converting it into an issue.
